.github/ISSUE_TEMPLATE/project-item-issue-template.yml

```
@@ -35,48 +33,48 @@ body:
 - type: checkboxes
   attributes:
     label: ToDo Items
-    description: Complete the following items before asking for a PR review
+    description: Complete the following items before requesting a PR review.
     options:
-    - label: Draft pull request created and linked to this issue
-    - label: Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
-    - label: Issue linked to the proper project
-    - label: Issue linked to proper milestone
+    - label: Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**).
+    - label: Issue linked to the correct project.
+    - label: Issue linked to the correct milestone.
+    - label: Draft pull request created and linked to this issue.
 
 - type: textarea
   attributes:
     label: Issue Dependencies
     placeholder: |
-      No Issue Dependencies
+      No Issue Dependencies Added.
       Example:
         - #10 Create new class
         - #20 Add unit tests to service
         - #30 Cleanup code in UI
     description: |
-      A list of issues that are dependencies of this issue and need to be finishes first
-      Use the '#\<number\>' syntax to link to an issue
+      A list of issues that are dependencies of this issue and need to be finished first.
+      Use the '#\<issue-number\>' syntax to link to an issue.
```

Should this say "link to an issue" or link an issue?
```suggestion
      Use the '#\<issue-number\>' syntax to link an issue.
```